src/index.html

```diff
@@ -1,13 +1,220 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a href="/" class="logo">
+              <img
+                src="./images/logo.svg"
+                class="logo__image"
+                alt="The MET"
+              >
+            </a>
+
+            <div class="header__icon-container">
+              <a href="tel:+1 212-535-7710" class="icon icon--phone"></a>
+
+              <a href="#menu" class="icon icon--menu-opener"></a>
+            </div>
```

Comment:
with icon you are creating a new block so if you are using header__icon-container for margin and position purposes you should rename it icon header__icon and you can rename the icon to icon__link and so icon--phone to icon__link--phone but -- is a modifier it is correct to use icon__link--menu-opener but I'm not sure about icon__link--phone